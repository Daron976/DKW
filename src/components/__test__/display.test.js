import renderer from 'react-test-renderer';
import Display from '../display';

describe('Display component', () => {
  test('display component renders as exepcted', async () => {
    const snapHome = renderer.create(<Display total="23" next="34" operation="+" />)
      .toJSON();
    expect(snapHome).toMatchSnapshot();
  });
});
