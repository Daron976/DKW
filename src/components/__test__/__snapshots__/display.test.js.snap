// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display component display component renders as exepcted 1`] = `
<p
  className="screen flex"
  data-testid="display"
>
  23
   
  +
   
  34
</p>
`;
